ocaml/xapi/pkg_mgr.mli

```diff
@@ -0,0 +1,29 @@
+val repoquery_sep : string
+
+type t = Yum | Dnf
+
+val active : unit -> t
```

Comment:
What is the purpose of `t` if it is never consumed? This is a code smell.
Below the many string parameters need documentation or labels.

Reply:
You can take it as an `Enum`, `active` requires it to tell the caller which one is active.